src/REPLMode.jl

```diff
@@ -171,9 +171,8 @@ end
 # packages can be identified through: uuid, name, or name+uuid
 # additionally valid for add/develop are: local path, url
 function parse_package(word::AbstractString; add_or_develop=false)::PackageSpec
-    word = replace(word, "~" => homedir())
```

Comment:
I wonder if we should keep this instead of using `expanduser`, to make this work on Windows too...

Reply:
I think `expanduser` is the right choice here. It seems that tilde has a distinct, special meaning in Windows (temporary files). 

I'm all for replacing `replace` :grin: